ACMILabs/interactive-label
# #63 Add GitHub Actions CI, and fix config.tmpl.env by removing comment

Resolves #63

Add Github Action to run tests.

### Acceptance Criteria
- [x] Add Github Actions to run Python tests
- [x] Add Github Actions to run Javascript tests

### Relevant design files
* None

### Testing instructions
1. See that this Action passes: https://github.com/ACMILabs/interactive-label/actions
2. Try running the development environment to see that still runs okay: `cp config.tmpl.env config.env`, edit config.env to include `XOS_PLAYLIST_ID=43`, and then `cd development` and `docker-compose up --build` and then visit http://localhost:8081

### DoD
For requester to complete:
